examples/nivisa_dmm_measurement/teststand_fixture.py

```diff
@@ -37,13 +36,16 @@ def update_pin_map(pin_map_path: str, sequence_context: Any) -> None:
     teststand_support.set_active_pin_map_id(pin_map_id)
 
 
-def create_nivisa_dmm_sessions(sequence_context: Any) -> None:
+def create_nivisa_dmm_sessions(sequence_context: Any, use_simulation: bool) -> None:
```

Comment:
@rfriedma I think you originally suggested the `USE_SIMULATION` constant. Is moving it to the TestStand file globals okay with you?

Reply:
I lean yes however, if a user sees "use_simulation" as a global on the sequence... I think it'd be expected that it would simulate everything. Would that be the case?

Reply:
@rfriedma, @bkeryan Now, I have added a new `_constants.py` file under `output_voltage_measurement` directory which holds the `USE_SIMULATION` boolean and is imported in `measurement.py`, `teststand_nidcpower.py` and `teststand_nivisa_dmm.py` and passed to the `create_session` method. 

Pros of this approach:
1. The existing user flow of updating the USE_SIMULATION bool in a .py is retained, just that now the user has to update the instrument mode in `_constants.py` instead of `_<instrument>_helpers.py`.
2. A `UseSimulation` boolean in teststand sequence is not required as it is handled using the same boolean in `_constants.py`. So, the user will update only one file to switch between hardware and simulation instead of updating in .py and in TestStand.

Cons:
1. A new `_constants.py` file is added.